pkg/api/vlabs/types.go

```diff
@@ -300,6 +300,14 @@ type KubernetesConfig struct {
 	CloudProviderRateLimit          bool              `json:"cloudProviderRateLimit,omitempty"`
 	CloudProviderRateLimitQPS       float64           `json:"cloudProviderRateLimitQPS,omitempty"`
 	CloudProviderRateLimitBucket    int               `json:"cloudProviderRateLimitBucket,omitempty"`
+	CustomFiles                     *[]CustomFile     `json:"customFiles,omitempty"`
```

Comment:
Could you please add a short description of `CustomFiles` in the docs? https://github.com/Azure/acs-engine/blob/master/docs/clusterdefinition.md is a good place to start